Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs

```diff
@@ -227,7 +227,7 @@ private void OnInteractUsing(EntityUid uid, ItemSlotsComponent itemSlots, Intera
                 if (slot.Item != null)
                     _handsSystem.TryPickupAnyHand(args.User, slot.Item.Value, handsComp: hands);
 
-                Insert(uid, slot, args.Used, args.User, excludeUserAudio: true);
+                Insert(uid, slot, args.Used, args.User, excludeUserAudio: default);
```

Comment:
`default` will become false here, it's a breaking change. You should just delete the entire `excludeUserAudio` argument here.

Also in other places.